source/client/output_formatter_impl.cc

```diff
@@ -143,7 +143,7 @@ std::string ConsoleOutputFormatterImpl::statIdtoFriendlyStatName(absl::string_vi
 }
 
 std::string JsonOutputFormatterImpl::formatProto(const nighthawk::client::Output& output) const {
-  return Envoy::MessageUtil::getJsonStringFromMessage(output, true, true);
+  return Envoy::MessageUtil::getJsonStringFromMessageOrDie(output, true, true);
```

Comment:
When does this die? Is that new,  and does it affect us?

Reply:
The change can be examined in this Envoy commit:

https://github.com/envoyproxy/envoy/commit/de8c5059baf575aaafdcc42572f7b019824987ba

The way I understand it, the rename doesn't introduce any new risk into our codebase. We may want to eventually switch to the version that returns the status and propagate that status back to the caller. But considering that would likely also result in a crash, just elsewhere in the stack - it probably is just cosmetic.

Please take a look at the linked commit and let me know if your interpretation differs.

Reply:
Agreed. Also, any crash here would be in a CLI, and not a service, so while not beautiful it's not as critical.

FWIW, there's one crash path in this areay that I know if today before the rename here raised awareness of other paths: the Fortio output formatter asserts when there's no global result. We could do better probably, as we could generate Fortio output which its UI will interpret as an error. Maybe it's worth filing an issue tagged as good-first-issue to put solidifying these paths as a whole on the (low priority) backlog.

Reply:
Thank you @oschaaf, agreed with your assessment about filing issue. If you get a few, would you mind filing it and including the additional background you have on those paths?